build.gradle

```diff
@@ -67,12 +72,12 @@ processResources {
 
     // Replace properties in info files
     from(sourceSets.main.resources.srcDirs) {
-        include '*.info'
-        expand 'version':project.version,'mc_version':config.mc_version
+        include 'mcmod.info'
+        expand 'version': project.version, 'mc_version': config.mc_version
```

Comment:
That was left over from trying to fix the version checking issue. I just copied it from CoFHCore I believe, since it was the only difference in the `mcmod.info` processing and I was wondering if it was causing the issues. I'll revert it.